### DISCUSSION TITLE
Zoho SalesIQ Chat Integration (salesiq_mobilisten)

### DISCUSSION THREAD

**Discussion by tryabp** [inspireui/support#5670] [Working on Features]
Hello everyone, 

Nowdays we ecommerce business owners are looking to interact with our customers 24x7 with use of technology and customised chat options. After reviewing many market leading tools i came to conclusion that for SME size business Zoho SalesIQ is the best tool you can get! And more to that **zoho also given plugin for flutter!** 

Zoho is having all in one CRM, customer support, sales, marketing tools that can be integrated into FluxStore app to make it more powerful in terms of customer service and engagement. So, i am posting this here to check how many of you are looking to solve the basic pain points of our business. 

With latest features like live translation, chatbots, automation, webhook and APi, possibilities with zoho salesiq are unlimited! Requesting you all to check the product using below link and upvote if you want to see this in fluxstore as well. 

PS: 
1. We got 3x more leads when we shifted to zoho salesIQ from traditional whatsapp chat on our website. 
2. We have integrated salesIQ chat link inside existing SmartChat by using direct link, its working in webview mode. 

I have already requested fluxstore team to look into salesIQ plugin. 

Links: 
Zoho SalesIQ : https://www.zoho.com/salesiq/
SalesIQ Flutter Plugin: https://pub.dev/packages/salesiq_mobilisten

**Comment by chungxon** [comment]
Hi, this feature is already supported on version 3.4.0 and later. Please check this video https://www.youtube.com/watch?v=a4nHGahHP08&t=3s
